src/EditorFeatures/Core.Wpf/Lightup/LightupHelpers.cs

```diff
@@ -311,11 +312,11 @@ public static Action<T> CreateActionAccessor<T>(Type? type, string methodName)
             throw new InvalidOperationException($"Type '{type}' is not assignable to type '{typeof(T)}'");
         }
 
-        var method = type.GetTypeInfo().GetDeclaredMethod(methodName);
-        if (method == null)
+        var method = type.GetTypeInfo().GetDeclaredMethods(methodName).Single(method =>
```

Comment:
❔ Was this change necessary? Eventually I'd like to recreate this with stronger validation (and better disambiguation), but as long as it gets the job done I wanted to keep the simpler form for now.

Reply:
Because on editor side, they introduced another overload of `GetSuggestionAsync`. So there are two methods now  and you will get exceptions when call `GetDeclaredMethod`

Reply:
(Agree we could make this better, but now since it's only accepting one parameter, this simple check seems fine)